tracim/tracim/lib/webdav/sql_resources.py

```diff
@@ -1005,6 +1005,40 @@ def move_file(self, destpath):
 
         transaction.commit()
 
+    def copyMoveSingle(self, destpath, isMove):
+        if isMove:
+            raise NotImplemented
+
+        new_file_name = None
+        new_file_extension = None
+
+        # Inspect destpath
+        if basename(destpath) != self.getDisplayName():
+            new_given_file_name = transform_to_bdd(basename(destpath))
+            new_file_name, new_file_extension = \
+                os.path.splitext(new_given_file_name)
```

Comment:
This should be refactored. Maybe the "move" method use a similar code block ?

Reply:
Yes, i do think a big refactor is needed, but i prefer to do this in the other PR.

Reply:
#548